meteor/client/lib/EditAttribute.tsx

```diff
@@ -229,7 +229,11 @@ const EditAttributeText = wrapEditAttribute(
 			this.handleEdit(event.target.value)
 		}
 		handleBlur(event) {
-			this.handleUpdate(event.target.value)
+			let value: string = event.target.value
+			if (value) {
+				value = value.trim()
+			}
```

Comment:
I think this might not always be desired. I'm thinking of some rare cases of people wanting to indent something with whitespaces etc. but I don't know if it's a real problem

Reply:
I agree with @LindvedKrvang. It is a good practice to trim input values. It will decrease the number of user mistypings.
Currently I am not aware of any input fields where indentation would matter.